cmd/car/testdata/script/root.txt

```diff
@@ -0,0 +1,15 @@
+car root ${INPUTS}/sample-v1.car
+cmp stdout v1root.txt
+
+car root ${INPUTS}/sample-wrapped-v2.car
+cmp stdout v2root.txt
+
+stop stdin test needs car fix
```

Comment:
note that the stdin variant of the command currently has issues. I want to get this command available in file mode today for users, so that work is filed as #282

Reply:
link the issue as a TODO?

Reply:
also, FYI, I usually do:

```
stop # explain the reason
```

same with skip etc